### ISSUE TITLE
Minor graphical bug (?) with CO face graphics in the battle animation pop-up

### ISSUE THREAD

**Issue by XnKradst** [Robosturm/Commander_Wars#254]
This is something I've noticed all the way back in beta release 5. I just didn't report it (until now) because I wasn't sure if it's intentional, or a bug.

It's hard to explain, but I'll try.

As you know, on the battle animation pop-up, the COs' faces are displayed in the corners, and their facial expressions change after the attack (and counter-attack), depending on how much damage it does.
The problem here is that only the attacker actually changes their expression, and not the defender. This means the attacking CO will show a happy expression when they deal significant damage to the defender (as expected), but the defender will keep whatever expression they started with, instead of changing to another one when it's appropriate.

I know it doesn't sound like a bug, and in fact, that's why I wasn't sure if this even was a bug.

But then I gave each player a second CO, and noticed it's reversed for the second COs. The second CO of the defender *does* change expressions, but *not* the second CO of the attacker.

So basically:
CO 1 of the attacker and CO 2 of the defender change expressions.
CO 2 of the attacker and CO 1 of the defender don't.

That sounds... unusual, so it's probably a bug?

**Comment by Robosturm** [comment]
Looked at the Code 100% a copy paste bug
